yellow-paper/docs/decentralization/actors.md

```diff
@@ -0,0 +1,134 @@
+
+These nodes are the sequencer and prover nodes. To function they must be connected to a Pruned Full Node, Full Node or Archive Node. 
+
+1. **Sequencer**: 
+   - Requires computational power to simulate blocks.
+   - Needs a robust network to coordinate proof generation and propagate blocks.
+2. **Provers**: 
+   - Have high-performance CPUs/GPUs and a lot of memory.
+   - Assists in proving zero-knowledge (zk) proofs for block validation.
+
+### Aztec Data Retrieval Capabilities:
+An Aztec node needs to be connected to an Ethereum L1 node to learn about the state of the system. Initially synching can be done this way, but due to the validity proofs, it will also support to download the state from a third party and then match it against the L1 state transitioner to validate that the state is correct. 
+
+As the Aztec node can be connected to different types of underlying network nodes, its ability to retrieve data will vary. Below we outline briefly what the capabilities are for each type of node:
+1. **Ethereum Light Node**:
+   - Relies on third party for getting L2 headers and block data.
+2. **Ethereum Pruned Full Node**:
+   - Can lookup recent L2 headers and blocks from own data.
+   - Relies on third party for getting historical blocks
+3. **Ethereum Full Node**:
+   - Can lookup L2 headers from own data.
+   - Can lookup recent L2 blocks from own data
+   - Relies on third party for getting historical blocks
+4. **Ethereum Archive Node**:
+   - Can lookup L2 headers and blocks from own data.
+
+### Actor Types
+
+1. **Retail**:
+   - Least knowledgeable and least interested in technical details.
+   - Prefers simplicity and ease of use.
+   - Likely to avoid running any node software or only running if bundled
+2. **Enthusiast**: 
+   - More knowledgeable and interested in technology than retail.
+   - Willing to invest time and resources but not at a professional level.
+3. **Power Users**: 
+   - Technically proficient and deeply engaged.
+   - Likely to have the resources and motivation to run more demanding nodes.
+4. **Developers**: 
+   - Highly knowledgeable with technical skills.
+   - Interested in detailed state and historical data for development and testing.
+5. **Self-Validators**: 
+   - Aim to participate actively in keeping the blockchain available.
+   - Interested in relying on their own nodes and resources (Archive Nodes).
+   - Likely to run nodes that contribute directly to the blockchain's operation, like Sequencers or Provers.
+6. **Institutions**: 
+   - Have significant resources (and potentially technical) capabilities.
+   - Interested in robust participation, potentially running Sequencer or Provers for financial gain.
+
+:::info State-Snapshots
+State snapshots is likely made and published by actors in the **Self-validator** or **Institution** category. These are actors that have the resources to run Archive Nodes and are interested in the historical data.
```

Comment:
```suggestion
State snapshots are likely made and published by actors in the **Self-validator** or **Institution** category. These are actors that have the resources to run Archive Nodes and are interested in the historical data.
```

This is placed a bit out-of-context here. Would expand on why we need state snapshots.

Reply:
State snapshot here refers to the publication of state at a specific point, e.g., you don't have the current, so it is just a snapshot of the state.  Related to your earlier comment.